asset/asset.go

```diff
@@ -694,6 +699,17 @@ type Asset struct {
 	GroupKey *GroupKey
 }
 
+// IsUnknownVersion returns true if an asset has a version that is not
```

Comment:
I think `IsUnknown` works better, since we use that as the condition to fail early where this function is called.

Otherwise we end up with something like `if !asset.IsKnownVersion() {` and inverting the result anyways.